protocol/synthetix/subgraph/tests/handleDelegationUpdated.ts

```diff
@@ -61,5 +62,18 @@ export default function test(): void {
   assert.fieldEquals('Position', `1-1-${address}`, 'collateral_amount', '10000');
   assert.fieldEquals('Vault', `1-${address}`, 'updated_at', (now + 1000).toString());
   assert.fieldEquals('Vault', `1-${address}`, 'updated_at_block', now.toString());
-  assert.fieldEquals('Vault', `1-${address}`, 'collateral_amount', '-5354');
+  assert.fieldEquals('Vault', `1-${address}`, 'collateral_amount', '10000');
+
+  const newDelegatioNUpdatedEvent3 = createDelegationUpdateEvent(
```

Comment:
odd casing... here and above too
```suggestion
  const newDelegationUpdatedEvent3 = createDelegationUpdateEvent(
```

Reply:
can probably just wrap it in `handleDelegationUpdated` and not add new variables